mods/default/functions.lua

```diff
@@ -152,7 +152,8 @@ end
 function default.grow_papyrus(pos, node)
 	pos.y = pos.y - 1
 	local name = minetest.get_node(pos).name
-	if name ~= "default:dirt_with_grass" and name ~= "default:dirt" then
+	if name ~= "default:dirt_with_grass" and name ~= "default:dirt" and
+			name ~= "group:sand" then
```

Comment:
* We don't want papyrus to grow on <i>sand</i>worm_spawners or on stuff registered by a tool<i>sand</i>blocks mod. Yes I made those examples up but we didn't ever do this in any other code, why do it here?

* Does this actually work? I don't think the name will be "group:sand" if it's in the sand group. Don't you need to look up the table of registered nodes for that?